_config.yml

```
@@ -1,2 +1,32 @@
+
+      - name: Lint Content Scripts
+        run: |
+          eslint src/content/**/*.js
+          eslint src/background/*.js
+
+  build-check:
+    runs-on: ubuntu-latest
+    needs: quality-checks
+    steps:
+      - uses: actions/checkout@v4
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)